src/main/java/mate/academy/spring/validation/MatchesPasswordValidator.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.spring.validation;
+
+import java.util.Objects;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.dto.request.UserRequestDto;
+
+public class MatchesPasswordValidator implements ConstraintValidator<RepeatedPasswordEquals,
+        UserRequestDto> {
+    @Override
+    public void initialize(RepeatedPasswordEquals constraintAnnotation) {
+    }
+
```

Comment:
good,let's make few changes 
6.Don’t override method initialize(), we don’t need it in validator class.